### DISCUSSION TITLE
Feature Request: Add GitHub Actor's Name as a GitHub Variable

### DISCUSSION THREAD

**Discussion by thisisthetechie** [community/community#109436] [Actions]
### Select Topic Area

Product Feedback

### Body

Please consider an update to the GitHub Actions runtime that could include `${{ github.actor_name }}` as an option (alongside `${{ github.actor }}`) so that we have the abilitity to use a proper name in labels and tags as opposed to the GitHub Username only.

**Comment by tr4l** [comment]
Also, I feel nervous that we don't have actor pronoun on the contexte.
Or maybe can we have a full name on github core ?
